### PR TITLE
LTD-2645: Create config data statically

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

Some changes required on API to support creating config data statically in lite_routing

### CODE REVIEW

**Review by kevincarrogan — APPROVED** [review]

**Comment by currycoder** [comment]
Presumably this will need to merge after https://github.com/uktrade/lite-routing/pull/16 and include a bump to the submodule SHA in order for these migrations to actually take effect

**Comment by saruniitr** [comment]
> Presumably this will need to merge after [uktrade/lite-routing#16](https://github.com/uktrade/lite-routing/pull/16) and include a bump to the submodule SHA in order for these migrations to actually take effect

yes that's correct. Once `lite-routing` PR is merged then I will update `lite-api`.

File: api/flags/tests/test_case_view_flag_ordering.py [hunk 1]
```diff
@@ -34,18 +37,14 @@ def test_gov_user_can_see_all_flags(self):
 
         self.organisation.flags.set([flag for flag in flags if flag.level == FlagLevels.ORGANISATION])
 
-        ordered_flags = get_ordered_flags(case, self.team)
```
Comment (saruniitr) [inline api/flags/tests/test_case_view_flag_ordering.py:37]: This test was checking the order of flags applied to the case.
The rules in migration started adding two new flags which affected the order.
I am not sure why the order is checked in this test which doesn't really matter as long as flags are present so updated the test to just check for presence of all flags.

File: api/teams/tests/factories.py [hunk 2]
```diff
@@ -3,12 +3,7 @@
 
 
 class DepartmentFactory(factory.django.DjangoModelFactory):
-    name = factory.Iterator(["HMRC", "MOD", "BEIS", "DIT", "NCSC"])
```
Comment (saruniitr) [inline api/teams/tests/factories.py:6]: it was complaining that department names already exists (as we create in migrations)

**Pull request merged**